tests/i18n/tests.py

```diff
@@ -1424,8 +1424,11 @@ def test_check_for_language(self):
         self.assertTrue(check_for_language('en'))
         self.assertTrue(check_for_language('en-us'))
         self.assertTrue(check_for_language('en-US'))
+        self.assertTrue(check_for_language('be'))
+        self.assertTrue(check_for_language('be@latin'))
```

Comment:
Could you add more tests? For example, "tr_TR.UTF-8", "tr_TR.UTF8" (without the dash), "de_DE.utf-8" (lowercase encoding), "sr_RS@latin", "sr_RS@12345".